packages/platforms/accton/x86-64/as5915-18x/platform-config/r0/src/lib/x86-64-accton-as5915-18x-r0.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+######################################################################
+#
+# platform-config for AS5915
+#
+######################################################################
+
+x86-64-accton-as5915-18x-r0:
+
+  grub:
+
+    serial: >-
+      --port=0x3f8
+      --speed=115200
+      --word=8
+      --parity=no
+      --stop=1
+
+    kernel:
+      <<: *kernel-4-14
+
+    args: >-
+      nopat
+      console=ttyS0,115200n8
+      intel_iommu=off
```

Comment:
Please also comment, why the IOMMU is disabled.

Reply:
Enabled.